src/main/java/org/myrobotlab/framework/Instantiator.java

```diff
@@ -18,7 +18,10 @@ static public Object getNewInstance(Class<?>[] cast, String classname, Object...
     try {
       return getThrowableNewInstance(cast, classname, params);
     } catch (Exception e) {
-      log.error("getNewInstance failed for {} {} {}", cast, classname, e.getClass());
+      // if we don't have gui it throw an error...
+      if (!classname.endsWith("Gui")) {
+        log.error("getNewInstance failed for {} {} {}", cast, classname, e.getClass());
```

Comment:
Something wonky here.   seems like this should be written like the following:

log.error("getNewInstance failed for {} {}", cast, classname, e);

Reply:
if "classname" is null.. this will cause an error..  please remove the  surrounding "if" statement.

Reply:
agree was tired... an idea to not get errors anymore about missing gui for service and filters?

Reply:
set to warn ?

Reply:
warn is fine..